### PR TITLE
Correct cross-entropy loss function

### LINKED ISSUE #1341: [Content] Incorrect cross_entropy_loss function
### Where does the issue happen?

https://github.com/Qiskit/platypus/blob/main/notebooks/quantum-machine-learning/vqc.ipynb

### What is the content issue?

In the cross_entropy_loss function, considering the relationship between `expected` and `p`, 
`return -(expected*np.log(p)+(1-expected)*np.log(1-p))` should be just `return -np.log(p)`.

### Additional information

This problem becomes apparent when the number of train and test data is increased to 50 and 20 respectively.

### PR DESCRIPTION
**Pull request by frankharkins**
Big thanks to @ry2sk and @adekusar-drl for help on this.

## Changes

- Corrected cross-entropy loss function (fixes #1341, see #1341 for details)
- Updated cell outputs to make sure the results match the new code
- Changed wording to reflect new results; looks like the optimizer performs better if we give it more time, so I've updated the text to reflect this.

## How to read this PR

Probably easiest to view each commit separately to see what I've changed, can also view a preview of the full page [here](https://platypus-pr-1417.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/).

### CODE REVIEW

**Review by vabarbosa — APPROVED** [review]
thank you
